javascript/javascript1/week2/flight-booking.js

```diff
@@ -0,0 +1,13 @@
+getFullname = (fullname1, fullname2, useFormalName) => {
+  if ((fullname1 || fullname2) === "") {
```

Comment:
what happens if I have an empty string like this *"                       "* and not *""*?

Reply:
just fixed it and commited